includes/acl/compression/impl/compact_constant_streams.h

```diff
@@ -110,7 +253,22 @@ namespace acl
 				const float constant_translation_threshold = settings.translation_format != vector_format8::vector3f_full ? desc.constant_translation_threshold : 0.0F;
 				const float constant_scale_threshold = settings.scale_format != vector_format8::vector3f_full ? desc.constant_scale_threshold : 0.0F;
 
-				if (is_rotation_track_constant(bone_stream.rotations, constant_rotation_threshold_angle))
+				if (
+
+#ifdef ACL_COMPRESSION_OPTIMIZED
+
+					bone_stream.is_rotation_constant || bone_range.rotation.is_constant() || ((constant_rotation_threshold_angle > 0.0F) &&
```

Comment:
Is the check for bone_stream_is_rotation_constant necessary? It should only be true here if we have a single sample (from the context init).

Reply:
I actually considered removing all of the constant and default processing from constant init, so we can do it all here and exert maximum control over tolerance tests(such as avoiding <= and < differences).  I left it alone just to keep the number of edits down.  Having said that, I can still remove the bone_stream.is_X_constant checks, because they're covered by the new ultra-precise bone_range.X.is_constant() calls anyhow.